src/bokeh/models/nodes.py

```diff
@@ -0,0 +1,214 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2023, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from typing import Any, Literal
+
+# Bokeh imports
+from ..core.properties import (
+    Either,
+    Enum,
+    Instance,
+    Int,
+    Required,
+    String,
+)
+from ..core.property.aliases import CoordinateLike
+from ..model import Model
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    "Indexed",
+    "Node",
+    "XY",
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+ImplicitTarget = Literal["viewport", "canvas", "plot", "frame", "parent"]
+
+class Coordinate(Model):
+    """
+    """
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+
+class XY(Coordinate):
+    """
+    A point in a Cartesian coordinate system.
```

Comment:
Have you enabled a code formatter locally? To date we have always started docstrings on the same line with a leading space
```
""" A point in a Cartesian coordinate system.
```
This seems to be different from what the broader community has settled on, so I'm inclined to consider changing. But we should at least strive for consistency regardless, so please open as issue to decide on one style and track updates for it.

Reply:
No, I don't use a formatter. It's just a drift in styling. I corrected all the cases.